botocore/handlers.py

```diff
@@ -423,6 +447,17 @@ def switch_host_with_param(request, param_name):
     ('before-call.s3.PutBucketCors', calculate_md5),
     ('before-call.s3.DeleteObjects', calculate_md5),
     ('before-call.s3.PutBucketReplication', calculate_md5),
```

Comment:
No, I believe these are all correct.  For example, PutBucketCORS documents Content-md5 as being a required parameter: http://docs.aws.amazon.com/AmazonS3/latest/API/RESTBucketPUTcors.html